test/integration_tests/test_container/build/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY deps/01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
+
+RUN apt-get -y update && \
+    apt-get -y install \
+        openjdk-11-jdk \
+        locales \
+        python3.8-venv \
+        python3-pip \
+        unixodbc \
+        unixodbc-dev \
+        curl \
+        git && \
+    locale-gen en_US.UTF-8 && \
+    update-locale LC_ALL=en_US.UTF-8 && \
+    apt-get -y clean && \
+    apt-get -y autoremove && \
+    ldconfig
+
+RUN curl -fsSL https://get.docker.com | bash
+
+RUN mkdir -p downloads/ODBC downloads/JDBC downloads/EXAplus
+RUN curl -s https://exasol-script-languages-dependencies.s3.eu-central-1.amazonaws.com/EXASOL_ODBC-7.0.11.tar.gz | tar -C downloads/ODBC --strip-components 1 -zxf -
```

Comment:
Can we upgrade the ODBC driver and ExaPlus to 7.1.x?

Reply:
I think, not for the moment, this might need changes in other repos. I will create a ticket for it.

Reply:
but maybe we can actually remove it here, because I think, it is not used. Let me check this.